pandas/core/generic.py

```diff
@@ -9136,7 +9136,8 @@ def resample(
         from pandas.core.resample import get_resampler
 
         if convention is not lib.no_default:
-            # TODO: Enforce in 3.0 (#55968)
+            # TODO: Plan to fix resampling with PeriodIndex instead (#55968)
+            # see https://github.com/pandas-dev/pandas/pull/62270#issuecomment-3262411892
```

Comment:
Should we also update the warning message with something similar? (that it won't be removed if it can be fixed)

Reply:
Seems to me that could encourage users to not use alternatives, which I think is the purpose of keeping the warning with the current behavior.